### PR TITLE
Updated to support XDG directory specification

### PR DESCRIPTION
**Pull request by bartenbach**
Updated the Makefile and README.

Should prefer $WEECHAT_HOME
If not set, attempt XDG directory
If not set, fallback to $HOME

### CODE REVIEW

**Review by Noskcaj19 — CHANGES_REQUESTED** [review]
The changes look good, except for the removal of some significant whitespace

File: README.md [hunk 1]
```diff
@@ -113,11 +113,11 @@ If you want a more irc-style interface, you can enable irc-mode:
 In irc-mode, weecord will not automatically "join" every Discord channel.  You must join a channel using the
 `/discord join <guild-name> [<channel-name>]` command.
 
-Watched channels:  
+Watched channels:
 You can use `/discord watch <guild-name> [<channel-name>]` to start watching a channel or entire guild.
 This means that if a message is received in a watched channel, that channel will be joined and added to the nicklist.
 
-Autojoin channels:  
+Autojoin channels:
```
Comment (Noskcaj19) [inline README.md:120]: These double spaces

File: README.md [hunk 2]
```diff
@@ -113,11 +113,11 @@ If you want a more irc-style interface, you can enable irc-mode:
 In irc-mode, weecord will not automatically "join" every Discord channel.  You must join a channel using the
 `/discord join <guild-name> [<channel-name>]` command.
 
-Watched channels:  
+Watched channels:
```
Comment (Noskcaj19) [inline README.md:116]: These double spaces

**Review by Noskcaj19 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (bartenbach) [reply README.md:116]: Whoops, that was a script to remove trailing whitespace. I can fix that.

**Comment by Noskcaj19** [comment]
Great, thanks!

**Pull request merged**